.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Feature request
+about: Suggest a feature for Sopel
+title: ''
+labels: enhancement
```

Comment:
We don't have this label; this should use 'Feature' instead.

```suggestion
labels: Feature
```

Reply:
Oop, apparently those were github's default labels, not ones copied from this repo to my fork.